readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -23,6 +23,7 @@ def setUp(self):
             slug='pip',
             repo='https://github.com/rtfd/readthedocs.org',
             privacy_level=PUBLIC,
+            external_builds_privacy_level=PUBLIC,
```

Comment:
Do we not want this to be the default for tests?

Reply:
Just being explicit with the default the .org gets, or test will break in .com. And we also have tests here that expect it to have PRs public, and being private will require an org in .com... soon we would be able to have all tests here.